pkg/virt-handler/vm.go

```diff
@@ -2816,6 +2815,10 @@ func (d *VirtualMachineController) vmUpdateHelperDefault(origVMI *v1.VirtualMach
 		if err := d.getMemoryDump(vmi); err != nil {
 			return err
 		}
+
+		if err := d.hotplugVirtioInterfaces(vmi); err != nil {
+			log.Log.Object(vmi).Error(err.Error())
```

Comment:
I think this should look like this:
`log.Log.Object(vmi).Reason(err).Error("")`

Reply:
I don't object, but it feels weird to print an empty string as the `Error` parameter.

Reply:
Up to you, it is a nit.

Reply:
Resolving then.